tests/online_migration/sle12_online_migration/snapper_rollback.pm

```diff
@@ -26,10 +26,22 @@ sub check_rollback_system() {
 sub run() {
     my $self = shift;
 
-    # login to snapshot and perform snapper rollback
-    assert_screen 'linux-login', 200;
+    # login to before online migration snapshot
+    # tty would not appear quite often after booting snapshot
+    # in this case try to select tty1 first then select root console
+    if (!check_screen('linux-login', 200)) {
+        for (1 .. 6) {
+            send_key "ctrl-alt-f1";
+            if (check_screen("tty1-selected", 5)) {
+                last;
+            }
+        }
+    }
+
     select_console 'root-console';
     wait_still_screen;
+
+    # perform snapper rollback
```

Comment:
stating the obvious. you don't need this comment. If you do, make it another commit as the change is unrelated.